catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/DorisTableOperationsIT.java

```diff
@@ -40,7 +40,7 @@ public class DorisTableOperationsIT extends TestDorisAbstractIT {
 
   private static final String databaseName = GravitinoITUtils.genRandomName("doris_test_db");
 
-  private static final long MAX_WAIT = 5;
+  private static final long MAX_WAIT = 30;
```

Comment:
Can you add some comments about the fact that Doris uses an asynchronous scheme change mechanism in the alter table and scheme operations? I'm afraid it's hard for users(Including me) to imagine this is the root cause.

Reply:
done